src/js/App.js

```diff
@@ -1,8 +1,31 @@
 import React, { Component } from 'react';
 import { View, StyleSheet } from 'react-native';
-import NavigationContainer from './Router';
+import NavigationContainer, { SCREENS } from './Router';
 import SwagLabsStatusBar from './components/StatusBar';
 import SplashScreen from 'react-native-splash-screen';
+import QuickActions from 'react-native-quick-actions';
+import { IS_IOS } from './config/Constants';
+
+QuickActions.setShortcutItems([
+	{
+		type: 'swagitems',
+		title: 'Open Swag Items',
+		subtitle: 'Open the Swag Items',
+		icon: IS_IOS ? 'Bookmark': 'bookmark',
```

Comment:
Why do this need to be have an uppercased first letter?

Reply:
This is needed for iOS icons, it uses the default iOS icons, the lowercase is for a custom Android icon

Reply:
Makes sense 🤔 

Perhaps it deserves a small comment on top of it?

Reply:
Will do